azext_iot/monitor/central_validator/validate_schema.py

```diff
@@ -0,0 +1,51 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from azext_iot.common.utility import ISO8601Validator
+from azext_iot.monitor.central_validator import utils
+from azext_iot.monitor.central_validator.validators import enum, geopoint, obj, vector
+
+iso_validator = ISO8601Validator()
+
+validation_function_factory = {
+    # primitive
+    "boolean": lambda schema, value: isinstance(value, bool),
```

Comment:
nit: Would be good to write a comment here or in validate with usage of the validate_function regarding lambda return values...something like "The return value of the lambda function is the value that this expression is evaluated to."

Reply:
this return type is likely to change in the next iteration, I'll add the comment there